app/javascript/components/Navbar.js

```diff
@@ -137,16 +199,35 @@ const NavbarComponent = ({items, openSearchMode}) => {
 };
 
 NavbarComponent.propTypes = {
-  items: PropTypes.array.isRequired,
-  openSearchMode: PropTypes.func.isRequired
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      controller: PropTypes.string,
+      action: PropTypes.string,
+      entry: PropTypes.string.isRequired,
+      path: PropTypes.string,
+      content: PropTypes.arrayOf(
+        PropTypes.shape({
+          title: PropTypes.string.isRequired,
+          slug: PropTypes.string.isRequired
+        })
+      ),
+      hasIcon: PropTypes.bool,
+      className: PropTypes.string
+    })
+  ).isRequired,
+  openSearchMode: PropTypes.func.isRequired,
+  active: PropTypes.shape({
```

Comment:
Is it possible to just do it looking only at url?

Reply:
Sure. The important is that they are mapped correctly at the items' entry level (i don't know the rails way™ to add the path there except hardcoded).

Reply:
Maybe active should in inside `items`

Reply:
https://github.com/Vizzuality/laws_and_pathways/blob/develop/app/views/layouts/cclow/geography.html.erb#L24

Reply:
If they have different urls, definitely.
What do you suggest on how to deal with them? Hardcoded? Should we put it in a helper/service or something?

Reply:
ahh, that would return link tag

Reply:
but there are other helpers https://github.com/comfy/active_link_to#helper-methods

Reply:
that could be added in layourt header.html.erb

Reply:
Yeah, due to how search behaves (overriding the nav) everything got out of rails context, so we need to somehow supply it via props (but if via path we don't need to supply the active one though).

Reply:
with those helpers is possible to just get is_active based on rails path.

Reply:
I was looking at `url_to` and `path_to` helper methods that are used internally in `link_to`. I'll give them a try.